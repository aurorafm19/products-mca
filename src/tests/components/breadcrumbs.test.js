import { render} from '@testing-library/react';
import Breadcrumbs from '../../components/breadcrumbs';

test('renders the component and shows just Home value', () => {
    const {container} = render(<Breadcrumbs/>);

    expect(container).toMatchSnapshot();
});
