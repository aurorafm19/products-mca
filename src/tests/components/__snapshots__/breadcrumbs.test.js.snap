// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component and shows just Home value 1`] = `
<div>
  <div
    class="breadcrumbs"
  >
    <a
      class="breadcrumbs"
      href="/"
    >
      Home
    </a>
    <div />
  </div>
</div>
`;
